language/english/admin.php

```diff
@@ -3,50 +3,46 @@
 /**
  * Language strings for admin page (en)
  *
- * @package chess
+ * @package    chess
  * @subpackage language
  */
 
 /**#@+
  * @ignore
  */
 
-define('_AM_CHESS_CONF',       'Chess');
+define('_AM_CHESS_CONF', 'Chess');
 
-define('_AM_CHESS_PREFS',      'Preferences');
+define('_AM_CHESS_PREFS', 'Preferences');
 define('_AM_CHESS_PREFS_DESC', 'Edit module preferences');
 
-define('_AM_CHESS_SUSPENDED_GAMES',     'Suspended games');
+define('_AM_CHESS_SUSPENDED_GAMES', 'Suspended games');
 define('_AM_CHESS_SUSPENDED_GAMES_DES', 'View games that have been suspended');
 
-define('_AM_CHESS_ACTIVE_GAMES',     'Active games');
+define('_AM_CHESS_ACTIVE_GAMES', 'Active games');
 define('_AM_CHESS_ACTIVE_GAMES_DES', 'View games that are active');
 
-define('_AM_CHESS_CHALLENGES',     'Challenges');
+define('_AM_CHESS_CHALLENGES', 'Challenges');
 define('_AM_CHESS_CHALLENGES_DES', 'View challenges');
 
-define('_AM_CHESS_GAME',           'Game');
-define('_AM_CHESS_CHALLENGE',      'Challenge');
-define('_AM_CHESS_VS',             'vs.');
-define('_AM_CHESS_CHALLENGED',     'challenged');
-define('_AM_CHESS_CREATED',        'created');
+define('_AM_CHESS_GAME', 'Game');
+define('_AM_CHESS_CHALLENGE', 'Challenge');
+define('_AM_CHESS_VS', 'vs.');
+define('_AM_CHESS_CHALLENGED', 'challenged');
+define('_AM_CHESS_CREATED', 'created');
 
-define('_AM_CHESS_WHEN_SUSPENDED',    'When suspended');
-define('_AM_CHESS_SUSPENDED_BY',      'Suspended by');
-define('_AM_CHESS_SUSPENSION_TYPE',   'Suspension type');
+define('_AM_CHESS_WHEN_SUSPENDED', 'When suspended');
+define('_AM_CHESS_SUSPENDED_BY', 'Suspended by');
+define('_AM_CHESS_SUSPENSION_TYPE', 'Suspension type');
 define('_AM_CHESS_SUSPENSION_REASON', 'Suspension reason');
 define('_AM_CHESS_SUSP_TYPE_ARBITER', 'Suspended by arbiter');
-define('_AM_CHESS_SUSP_TYPE_PLAYER',  'Arbitration requested');
+define('_AM_CHESS_SUSP_TYPE_PLAYER', 'Arbitration requested');
 
-define('_AM_CHESS_ARBITRATE_SUBMIT',   'Arbitrate');
+define('_AM_CHESS_ARBITRATE_SUBMIT', 'Arbitrate');
 
 define('_AM_CHESS_NO_SUSPENDED_GAMES', 'No suspended games');
-define('_AM_CHESS_NO_ACTIVE_GAMES',    'No active games');
-define('_AM_CHESS_NO_CHALLENGES',      'No challenges');
+define('_AM_CHESS_NO_ACTIVE_GAMES', 'No active games');
+define('_AM_CHESS_NO_CHALLENGES', 'No challenges');
 
-define('_AM_CHESS_UNKNOWN_USER',       '*unknown*');
-define('_AM_CHESS_ERROR',              '*error*');
-
-/**#@-*/
-
-?>
\ No newline at end of file
+define('_AM_CHESS_UNKNOWN_USER', '*unknown*');
+define('_AM_CHESS_ERROR', '*error*');
```

Comment:
Need closing on phpDoc comment so that these are not shown (ignored) when creating phpDoc
```suggestion
define('_AM_CHESS_ERROR', '*error*');
/**#@-*/
```